README.md

```diff
@@ -1,18 +1,26 @@
 # BigImageViewer
 
-[ ![Download](https://api.bintray.com/packages/piasy/maven/BigImageViewer/images/download.svg) ](https://bintray.com/piasy/maven/BigImageViewer/_latestVersion)
+[![Download](https://api.bintray.com/packages/piasy/maven/BigImageViewer/images/download.svg)](https://bintray.com/piasy/maven/BigImageViewer/_latestVersion)
```

Comment:
I believe a lot of the restructuring especially the getting started part make it a lot more clearer. Every major project has it.